Header header
geometry_msgs/Pose pose
geometry_msgs/Twist twist
string flight_command # possible commands: ARM, POSTITION, VELOCTIY, OFFBOARD